docs/frontend-system/architecture/05-extension-overrides.md

```diff
@@ -10,38 +10,93 @@ description: Frontend extension overrides
 
 ## Introduction
 
-<!--
+An extension override is a building block of the Frontend System that allows you to programmatically override app or plugin extensions anywhere in your application. Since the entire application is built mostly out of extensions from the bottom up, this is a powerful feature. You can use it for example to provide your own app root layout, to replace the implementation of a Utility API with your a custom one, to override how the catalog page renders itself, and much more.
 
-Introduce extension overrides and how they can be shipped in separate packages but also internally within a project too.
+Note that in general, most features should have a good level of customizability built into themselves, so that users do not have to leverage extension overrides to achieve common goals. A well written feature often has app-config settings, or uses extension inputs for extensibility where applicable. An example of this is the search plugin which allows you to provide result renderers as inputs rather than replacing the result page wholesale just to tweak how results are shown. Adopters should be taken advantage of those when possible, and only use extension overrides when it's necessary to entirely replace the extension. Check the respective extension documentation if available for guidance.
 
-Talk about how extension overrides
+## Overriding App Extensions
 
--->
+If you want to override an app extension, you will need to create a new extension to replace the existing one and add it to the list of overridden features. The steps are:
 
-## Creating a Extension Override
+1. Create your extension overrides
 
-<!--
+In this example below, we are going to create custom extensions for the app light and dark themes:
```

Comment:
In general, if you indent _everything_ by three spaces so that it's in line with the "Create your extension overrides" text above, then it'll be made part of the numbered list item as expected.

```
1. Foo

Bar
```

becomes 

%%%%%%%%%%%%
1. Foo

Bar

%%%%%%%%%%%%

but

```
1. Foo

   Bar
```

becomes

%%%%%%%%%%%%
1. Foo

   Bar

%%%%%%%%%%%%

Reply:
I restructured the page, thanks for the tip 🎉

Reply:
I also added a `Creating Orphan Extensions` sub-section, could you please take a look @freben ?